apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/template-update-notice/index.js

```diff
@@ -0,0 +1,28 @@
+/* global fullSiteEditing */
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { render } from '@wordpress/element';
+import { Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+domReady( () => {
+	if ( 'wp_template_part' !== fullSiteEditing.editorPostType ) {
+		return;
+	}
+	const element = document.createElement( 'div' );
+	element.classList.add( 'wp-block-a8c-template-edit-warning' );
+	document.getElementsByClassName( 'edit-post-layout__content' )[ 0 ].prepend( element );
+	render(
+		<Notice isDismissible={ false }>
```

Comment:
Do we want it to be non dismissible? (just curious, not sure what the spec is)

Reply:
@shaunandrews  - your designs didn't show any close 'x' in this notice so I read that to mean it wasn't dismissible  - can you just confirm that is the case before I go ahead and merge this please?